plugins/woocommerce-admin/client/customer-effort-score-tracks/customer-effort-score-tracks.js

```diff
@@ -207,12 +221,16 @@ export default compose(
 		const allowTracking = allowTrackingOption === 'yes';
 
 		const resolving =
-			isResolving( 'getOption', [ SHOWN_FOR_ACTIONS_OPTION_NAME ] ) ||
+			! hasFinishedResolution( 'getOption', [
+				SHOWN_FOR_ACTIONS_OPTION_NAME,
+			] ) ||
 			storeAgeInWeeks === null ||
-			isResolving( 'getOption', [
+			! hasFinishedResolution( 'getOption', [
 				ADMIN_INSTALL_TIMESTAMP_OPTION_NAME,
 			] ) ||
-			isResolving( 'getOption', [ ALLOW_TRACKING_OPTION_NAME ] );
+			! hasFinishedResolution( 'getOption', [
+				ALLOW_TRACKING_OPTION_NAME,
+			] );
```

Comment:
Made use of `hasFinishedResolution` to fix a bug where if one of the above options was being retrieved again `isResolving` would be set to `true` again, which in turn set `resolving` to true and in turn caused this component to `return null`.
Once the resolving finished it would render the `CustomerEffortScore` again and in turn trigger a new notice, something we don't want.

Reply:
Nice catch!